test/RibbonTreasuryVault.ts

```diff
@@ -578,7 +618,7 @@ function behavesLikeRibbonOptionsVault(params: {
         secondOptionExpiry = moment(latestTimestamp * 1000)
           .endOf("month")
           .add(chainId === CHAINID.AVAX_MAINNET ? 0 : 1, "weeks")
-          .add(1, "month")
+          .add(asset === SPELL_ADDRESS[chainId] ? 0 : 1, "month")
```

Comment:
since block timestamp we use for SPELL is awkward date in month (July 14th), our math for mid-month gets second option expiry wrong - this is workaround